.gitlab/JOBOWNERS

```diff
@@ -41,10 +41,10 @@ agent_suse*                          @DataDog/agent-build-and-releases
 iot_agent_suse*                      @DataDog/agent-build-and-releases
 dogstatsd_suse*                      @DataDog/agent-build-and-releases
 agent_dmg*                           @DataDog/agent-build-and-releases
+*oci*                                @DataDog/agent-build-and-releases
 windows_msi*                         @DataDog/windows-agent
 
 # Packaging
-agent_oci                            @DataDog/agent-build-and-releases
 updater_deb-*                        @DataDog/agent-build-and-releases
```

Comment:
those would need to be renamed to `installer_*` (although it's not strictly related to your PR)